spotipy/util.py

```diff
@@ -107,6 +116,16 @@ def prompt_for_user_token(
 
 
 def get_host_port(netloc):
+    """Split the network location string into host and port.
+
+    Args:
+        netloc: A string representing the network location.
+
+    Returns:
+        A tuple where the first element is the host as a string,
+        and the second element is the port as an integer.
+        None if no port is specified.
```

Comment:
The same here. Please make sure to use the same format like in our client.py file.

Reply:
I have reverted it the same way like the one above.